src/Dhall/Core.hs

```diff
@@ -1130,6 +1142,7 @@ normalize e = case e of
             App NaturalIsZero (NaturalLit n) -> BoolLit (n == 0)
             App NaturalEven (NaturalLit n) -> BoolLit (even n)
             App NaturalOdd (NaturalLit n) -> BoolLit (odd n)
+            -- What does this do? (TODO)
```

Comment:
I am not quite sure what the rule for `ListBuild` does here in the normalization case, could you maybe explain a little bit?  For example, what is `Var "Cons"` referring to?